extensions/autoPepSIRF.py

```diff
@@ -254,6 +259,20 @@ def main():
             csD = io.fileDictFull(args.zscore, valType="float", rowNames=True)
             for r1, r2 in pD.items():
                 scatter(csD[r1], r1, csD[r2], r2, "zRepScatters/%s_%s_Z.%s" % (r1, r2, args.scatterFormat), plotLog=False)
+                
+        #Enriched peptides scaterplot generation
+        if args.enrichedScatters and args.colsum:
+            cmd = ('python3 %s -d %s -e %s --enrExt _enriched.txt -x NegativeControl -o %s/enrichedScatterplots --plotLog 1' % (args.enrichedScatters, args.colsum, enrDir, enrDir))
+            
+            if args.negNormMat:
+                cmd += (' --negMatrix %s' % args.negNormMat)
+                
+            if args.negative_id:
+                cmd += (' -i %s' % args.negative_id)
+            elif args.negative_names:
```

Comment:
Let's go ahead and make this an if statement instead of elif. This way, we pass through everything provided to autoPepSIRF, and lead it to the enrichedScatterplot script to decide how to deal with it.